lib/index.js

```diff
@@ -112,13 +113,22 @@ function Server (opts, cb) {
 
   this.dataDir = opts.dataDir || opts.db || path.join(__dirname, '/../data');
 
-
   mkdirp(this.dataDir, function (err) {
     if (err) console.error(err);
-    else self.createDB(cb);
+    else {
+      self.createDB(cb);
+      /*
+       * Set up the routes of the app
+       */
+      if (opts.defaultRoutes !== false) {
+        apiV2.install(self);
+        accounts.install(self);
+        sheets.install(self);
+        sessions.install(self);
```

Comment:
To ensure that our models are created by the time we set up the routes, I moved the route initializations up here. I checked that it doesn't interfere with anything, but let me know if there is a better spot for it.